openedx/core/djangoapps/discussions/models.py

```diff
@@ -25,6 +26,8 @@
 DEFAULT_PROVIDER_TYPE = 'legacy'
 
 
+DocumentLinks = namedtuple('DocumentLinks', ['learn_more', 'configuration_documentation', 'documentation','accessibility_documentation', 'email_id'])
```

Comment:
FAILURE: Too many PEP 8 violations. Number of PEP 8 violations: 2

Violations:
./openedx/core/djangoapps/discussions/models.py:29:106: E231 missing whitespace after ','
./openedx/core/djangoapps/discussions/models.py:31:1: E302 expected 2 blank lines, found 1